recipe/meta.yaml

```diff
@@ -12,17 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
     - setuptools
   run:
     - python
-    - numpy >=1.8.1
+    - numpy
```

Comment:
This designation should stay in place.

Reply:
`conda-forge` builds with a `numpy >=1.9` so this is "OK." However, if you want to be friendly with people fetching this recipe and building it in other setups keep that is OK.